src/users/email_verifications/routes.rs

```diff
@@ -0,0 +1,97 @@
+use std::sync::Arc;
+
+use axum::extract::Path;
+use axum::routing::post;
+use axum::{extract::State, Router};
+use chrono::{Duration, Utc};
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+use uuid::Uuid;
+
+use crate::schema::users;
+use crate::InnerAppState;
+use crate::{auth::AuthExtractor, schema::email_verifications, users::models::UserRole, AppState};
+
+use super::{models::EmailVerification, EmailVerificationError};
+
+pub fn email_verification_router() -> Router<AppState> {
+    Router::new()
+        .route("/email_verification", post(create_email_verification))
+        .route("/confirm_email/:verification_id", post(confirm_email))
```

Comment:
This should follow the rest conventions for consistency 

\- for space instead of _

```suggestion
        .route("/email-verification", post(create_email_verification))
        .route("/confirm-email/:verification_id", post(confirm_email))
```